### ISSUE TITLE
Feature Request - Support for SMS:2D Mesh Files *.2dm

### ISSUE THREAD

**Issue by Brent-Edwards** [nschloe/meshio#496]
Any chance that you could add support for this format - [xmswiki.com/wiki/SMS:2D_Mesh_Files_*.2dm](url)?

Thanks!

**Comment by Brent-Edwards** [comment]
Thanks for the reply. Can MDAL be used to translate between the different formats supported by the mesh provider?

**Comment by saberraz** [comment]
Yes...there is meal_translate (similar to gdal translate) utility which we have added recently. 

Best to move the discussion there!

**Comment by Brent-Edwards** [comment]
Ok found it - [https://github.com/lutraconsulting/MDAL/issues/167](url).

Thanks!
